src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
```

Comment:
The check for property presence should be done using `Object.hasOwnProperty()` method to avoid issues with inherited properties. Directly using `in` might lead to unexpected behavior if the object has inherited properties with the same name as the values you are checking.